roles/db-copy/tasks/active-copy.yml

```diff
@@ -225,6 +243,13 @@
       become_user: "{{ oracle_user }}"
       no_log: true
```

Comment:
Now that we don't have a password on the command line, can we get away without no_log?
The RMAN command does a lot of work, and it's tough to troubleshoot without the output.  (It does create a log, but doesn't do anything with it AFAICT)